android/engine/src/test/resources/plans/child-immunization-schedule/plandefinition.json

```diff
@@ -8,105 +8,14 @@
       "title": "Child Immunization CarePlan",
       "status": "active",
       "description": "This action will assess careplan on registration to init careplan",
-      "kind": "CarePlan",
+      "kind": "Task",
       "dynamicValue": [
-        {
-          "path": "title",
-          "expression": {
-            "language": "text/fhirpath",
-            "expression": "%rootResource.title"
-          }
-        },
         {
           "path": "description",
           "expression": {
             "language": "text/fhirpath",
             "expression": "%rootResource.description"
           }
-        },
-        {
-          "path": "instantiatesCanonical",
-          "expression": {
```

Comment:
instantiatesCanonical is not a string anymore. All PlanDefinitons must change to CannonicalType, assuming instantiatesCanonical is writeable from the action.